src/pages/explore/dfi/_components/GetDFISection.tsx

```diff
@@ -0,0 +1,130 @@
+import classNames from "classnames";
+import Link from "next/link";
+import { useEffect, useState } from "react";
+import { Container } from "@components/commons/Container";
+import { SectionTitle } from "@components/commons/SectionTitle";
+import { useTranslation } from "next-i18next";
+import BigNumber from "bignumber.js";
+import { NumericFormat } from "react-number-format";
+import CoinGeckoIcon from "@components/icons/assets/exploreGetDFI/CoinGeckoIcon";
+import CoinMarketCapIcon from "@components/icons/assets/exploreGetDFI/CoinMarketCapIcon";
+import GetDFISectionExchanges from "./GetDFISectionExchanges";
+
+export default function GetDFISection() {
+  const [coinMarketCapPrice, setCoinMarketCapPrice] = useState<string>("0");
```

Comment:
let's not display 0 as default, to prevent unnecessary drama lol, maybe `N/A`, same for coingecko

Reply:
i mean to display `N/A` in the UI, we can store undefined as the state

Reply:
> i mean to display `N/A` in the UI, we can store undefined as the state

Just saw this last comment, I think I did it this way 👍